### ISSUE TITLE
type-script api error

### ISSUE THREAD

**Issue by zhex900** [ryanto/next-deploy-notifications#5]
```
// pages/api/has-new-deploy.ts

import { APIRoute } from 'next-deploy-notifications';

export default APIRoute.configure({
  version: () =>
    process.env.NEXT_BUILD_ID || process.env.VERCEL_GIT_COMMIT_SHA || '',
});

```
This will cause the endpoint fail.

If I change the import to `import { APIRoute } from 'next-deploy-notifications/api' 

A type error occurs

 `Cannot find module 'next-deploy-notifications/api' or its corresponding type declarations.`

How do I fix it?

**Comment by ryanto** [comment]
Thanks for reporting! I'll get a fix for this out soon.

As a quick fix you can make the file a regular old Javascript file and it should fix the error.

**Comment by ryanto** [comment]
Ok I published version `0.1.5` that should fix this.

Just to note it looks like this may have been a bug/feature that was recently addressed in typescript. Make sure you're on the latest version of typescript (4.7.x).

Let me know how it goes!

**Comment by zhex900** [comment]
```
Type error: Cannot find module 'next-deploy-notifications' or its corresponding type declarations.

> 1 | import { useHasNewDeploy } from 'next-deploy-notifications';
```
This is breaking now

**Comment by ryanto** [comment]
Ugh sorry about that!

I just released `0.1.6` that should fix this.

**Comment by ryanto** [comment]
Hey I'm going to close this, but let me know if you're still running into issues.
